tests/companies/test_enums.py

```diff
@@ -0,0 +1,25 @@
+from itou.companies.enums import CompanyKind
+
+
+class TestCompanyKind:
+    def test_siae_choices(self):
```

Comment:
```suggestion
    def test_siae_kinds(self):
```
Sinon je m'attend à ce qu'on test `CompanyKind.siae_choices()` ;)